File: RoRServer/src/main/java/models/game/Player.java
package models.game;

import communication.MessageInformation;
import models.base.ModelBase;

import java.util.UUID;

public class Player extends ModelBase {
    private String name;
    private boolean isHost;

    public Player(String sessionName, String name, UUID id, boolean isHost) {
    	super(sessionName);
        this.name = name;
        this.setId(id);
        this.isHost = isHost;
    }
    
    public String getDescription() {
        return name;
    }

    public void setSessionName(String name) {
        this.name = name;
    }
    
    public boolean getIsHost() {
    	return isHost;
    }
}

File: RoRClient/Models/Game/Player.cs
using RoRClient.Models.Base;
using System;
using System.Collections.ObjectModel;

namespace RoRClient.Models.Game
{
    public class Player: ModelBase
    {
        public Player(Guid id, string name)
        {
            this.name = name;
            this.id = id;
        }

        public Player(Guid id, string name, bool isHost) : this(id, name)
        {
            this.isHost = isHost;
        }

        private string name;
        public String Name
        {
            get
            {
                return name;
            }
        }

        private bool isHost;
        public bool IsHost
        {
            get
            {
                return isHost;
            }
        }
    }
}
